packages/legend-graph/src/graph/PureModel.ts

```diff
@@ -581,6 +587,34 @@ export class PureModel extends BasicModel {
     return element;
   }
 
+  getElementOrigin(
+    elementPath: string,
+    element?: PackageableElement | undefined,
+  ): string {
+    let _origin = '';
+    let packageableElement: PackageableElement;
+    if (element) {
+      packageableElement = element;
+    } else {
+      packageableElement = guaranteeNonNullable(
+        this.getNullableElement(elementPath),
+      );
+    }
+    if (isSystemElement(packageableElement)) {
+      _origin = ROOT_PACKAGE_NAME.SYSTEM.slice(0, -5);
```

Comment:
this looks really bizzare, why don't we just hard-code text here, after all, this method is basically a `pretty print` of element origin.